src/dotnet-install.ps1

```diff
@@ -938,9 +963,9 @@ $ScriptName = $MyInvocation.MyCommand.Name
 
 if ($DryRun) {
     Say "Payload URLs:"
-    Say "Primary named payload URL: $DownloadLink"
+    Say "Primary named payload URL: ${DownloadLink}${FeedCredential}"
```

Comment:
I don't think we are allowed to print feed credentials.

Reply:
This is not showed in logs, even verbose, this is the dry-run. FeedCredential is already logged in dry run few lines below in the repeatable command. It makes sense to me to have FeedCredential as a part of URL here, so the person can actually have a downloadable link. But this may be debatable.

Reply:
Proper general guidance by @vlada-shubina .  In this dry-run's use case, the URL should be actionable.